go/libraries/doltcore/sqle/dsess/session_state_adapter.go

```diff
@@ -121,10 +121,6 @@ func (s SessionStateAdapter) AddRemote(remote env.Remote) error {
 		return env.ErrRemoteAlreadyExists
 	}
 
-	if strings.IndexAny(remote.Name, " \t\n\r./\\!@#$%^&*(){}[],.<>'\"?=+|") != -1 {
-		return env.ErrInvalidBackupName
```

Comment:
I don't know why this was in here, but its technically not a Backup error. This function is used by other procedures.

Reply:
I can add a test for it, but I don't think they were checking.